test/test_schemas.py

```diff
@@ -0,0 +1,57 @@
+from boundary_layer.schemas.dag import BatchingSchema
```

Comment:
Let me know if there's a better place to put this.

Reply:
oh ya I wouldn't bet on us having perfect fidelity with this, but we have tried to align the locations of files in `test/` with the directory structure that is used in the main package.  So could you please put this file into a directory `test/schemas` ?